crates/sui-source-validation/src/lib.rs

```diff
@@ -86,8 +90,12 @@ impl<'a> BytecodeSourceVerifier<'a> {
         compiled_package: &CompiledPackage,
         root_on_chain_address: AccountAddress,
     ) -> Result<(), SourceVerificationError> {
-        self.verify_package(compiled_package, false, Some(root_on_chain_address))
-            .await
+        self.verify_package(
+            compiled_package,
+            /* do not verify_deps */ false,
```

Comment:
nit: The usual pattern is just to label it with the parameter name, to make it kind of like a keyword parameter.  There's a risk that someone might read this line as a double negative.

```suggestion
            /* verify_deps */ false,
```

I don't feel particularly strongly about this though!